tests/integration/services/user.test.ts

```diff
@@ -182,6 +183,107 @@ describe('user service', () => {
                 pushNotificationToken: 'ckniwoaicoska',
             });
         });
+
+        it('authentication should return an error when trying to create a new account with an already used phone', async () => {
+            const firstRandomWallet = ethers.Wallet.createRandom();
+            const secondRandomWallet = ethers.Wallet.createRandom();
+            const firstAddress = await firstRandomWallet.getAddress();
+            const secondAddress = await secondRandomWallet.getAddress();
+            const phone = faker.phone.phoneNumber();
+
+            await UserService.authenticate({
+                address: firstAddress,
+                trust: {
+                    phone,
+                },
+            });
+            let error: any;
+            await UserService.authenticate({
+                address: secondAddress,
+                trust: {
+                    phone,
+                },
+            }).catch(err => {
+                error = err;
+            });
+
+            expect(error.message).to.equal('phone associated with another account')
+        });
+
+        it('authentication should inactivate the old account and create a new', async () => {
+            const firstRandomWallet = ethers.Wallet.createRandom();
+            const secondRandomWallet = ethers.Wallet.createRandom();
+            const firstAddress = await firstRandomWallet.getAddress();
+            const secondAddress = await secondRandomWallet.getAddress();
+            const phone = faker.phone.phoneNumber();
+
+            await UserService.authenticate({
+                address: firstAddress,
+                trust: {
+                    phone,
+                },
+            });
+
+            const loadUser = await UserService.authenticate({
+                address: secondAddress,
+                trust: {
+                    phone,
+                },
+                overwrite: true,
+            });
+
+            const findUser = await sequelize.models.UserModel.findOne({
```

Comment:
you can do
```suggestion
            const findUser = await models.user.findOne({
```
it's easier to find issues if we change either the name or any parameter.